docs/source/download.rst

```diff
@@ -1,64 +1,107 @@
-=============
-Obtain Devito
-=============
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_.
+For those looking to use Devito alongside other packages as part of a project, we support
+`pip installation`_. When developing for Devito, you might benefit from
+following the `conda installation`_ .
 
-There are three main approaches to installing Devito. For those looking for the
-least-friction way to try Devito, we recommend the Docker route. For those
-looking to use Devito alongside other packages as part of a project, we support
-pip installation. If you're looking to develop for Devito, you might benefit from
-using the included conda environment that includes all the bells and whistles we
-recommend when developing for Devito.
 
 Docker installation
 -------------------
+
 You can install and run Devito via Docker_:
 
 .. _Docker: https://www.docker.com/  
 
+There are several "tags" for the devito docker images. We provide `Docker for CPUs`_
```

Comment:
Maybe "devito" and "docker" should be capitalized to here to be consistent with capitalization later.